decentraland/unity-explorer
# feat: shaders notification text

## What does this PR change?

First start-up takes much time (40-50 seconds), let's notify user to don't worry. During the splash screen the information text will be shown only on first launch (or any other situation when shaders are removed).

Works only on windows, because mac doesn't require shaders compilation.

## How to test the changes?

1. Launch the explorer 
2. Watch text
3. Relaunch it
4. Should start without text

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
